pubsub/gossipsub/gossipsub-v1.1.md

```diff
@@ -470,6 +471,20 @@ if invalidMessageDeliveries < DecayToZero {
 
 `TBD`: We are currently developing multiple types of simulations that will inform us on how to best recommend tunning the Scoring function. We will update this section once that work is complete
 
+#### Extended Validators
+
+The pubsub subsystem incorporates application-specific message validators so that the application can
+signal invalid message delivery, and trigger the P₄ penalty.
+However, It is possible to have circumstances where a message should not be delivered to the application
+or forwarded to the network, but without triggering the P₄ penalty.
+This might arise in the case of duplicate beacon messages or while an application is syncing its blockchain, in which case it would be unable to ascertain the validity of new messages.
```

Comment:
```suggestion
A known use-case where this need exists is in the case of duplicate beacon messages or while an application is syncing its blockchain, in which case it would be unable to ascertain the validity of new messages.
```